### PR TITLE
IMMUTANT-568: no concurrent exec job fix

### PR DESCRIPTION
**Pull request by instilled**
this solves https://issues.jboss.org/browse/IMMUTANT-568

### CODE REVIEW

File: modules/scheduling/src/main/java/org/projectodd/wunderboss/scheduling/Scheduling.java [hunk 1]
```diff
@@ -35,6 +35,7 @@
         public static final ScheduleOption LIMIT     = opt("limit", ScheduleOption.class);
         public static final ScheduleOption UNTIL     = opt("until", ScheduleOption.class);
         public static final ScheduleOption SINGLETON = opt("singleton", true, ScheduleOption.class);
+        public static final ScheduleOption ALLOW_CONCURRENT_EXEC = opt("allow-concurrent-exec?", true, ScheduleOption.class);
```
Comment (tobias) [inline modules/scheduling/src/main/java/org/projectodd/wunderboss/scheduling/Scheduling.java]: The option name needs to be `"allow_concurrent_exec"` so we can use the option from TorqueBox. Immutant will automatically convert `:allow-concurrent-exec?` to `"allow_concurrent_exec"`.

**Comment by tobias** [comment]
Thanks! Looks good. If you can change that option name, I'll merge.

**Comment by instilled** [comment]
done!

**Comment by tobias** [comment]
Thanks! I'll merge the Immutant PR as well, and add a test there.

**Comment by instilled** [comment]
Thanks Toby.

**Pull request merged**